### PR TITLE
Adding the spec file (SOFTWARE-3102)

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: rpm/repo-update-cadist.spec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Release:   1%{?dist}
+License:   Apache 2.0
+Group:     Grid
+URL:       https://github.com/opensciencegrid/repo-update-cadist
+BuildArch: noarch
+
+
+Source0:   %{name}
+
+BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```
Comment (matyasselmeci) [inline rpm/repo-update-cadist.spec]: BuildRoot no longer necessary for EL 6+

File: rpm/repo-update-cadist.spec [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+%description
+%{summary}
+
+%install
+mkdir -p $RPM_BUILD_ROOT%{_bindir}
+install -pm 755 %{SOURCE0}  $RPM_BUILD_ROOT%{_bindir}/
+
+%clean
```
Comment (matyasselmeci) [inline rpm/repo-update-cadist.spec]: %clean section no longer necessary for EL6+

**Review by matyasselmeci — CHANGES_REQUESTED** [review]
The BuildRoot header and the %clean section are no longer necessary on EL6+ and should be removed. @timtheisen, since you put together the spec file, you should be aware of this too.

**Comment by matyasselmeci** [comment]
You don't need to do a rebuild after you make this change since it doesn't affect the functionality or the finished product.

**Comment by efajardo** [comment]
@matyasselmeci I made the changes you requested.

**Comment by efajardo** [comment]
@matyasselmeci I made the changes you requested.

**Comment by matyasselmeci** [comment]
By "EL6+" I meant EL6 or greater. Neither EL6 nor EL7 need those parts, and we've stopped supporting EL5. So you should delete those parts entirely.

**Comment by efajardo** [comment]
@matyasselmeci  I added the changes you requested.

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**